src/mudlet-lua/lua/Other.lua

```diff
@@ -418,6 +418,10 @@ function speedwalk(dirString, backwards, delay, show)
   local walkdelay = delay
   if show ~= false then show = true end
   local walklist = {}
+  local long_dir = {north = 'n', south = 's', east = 'e', west = 'w', up = 'u', down = 'd'}
```

Comment:
Are `in` and `out` an issue to be handled here as well? :confused:

Reply:
In and out are managed later in the function in how it parses dirString, using this pattern: ([0-9]*)([neswudio][ewnu]?t?)
As you can see, it will properly capture "in" and "out", as well as shortened versions of them, but it won't handle any of the other long directions.

Reply:
Okay, I wasn't sure but if you are that is good enough for me.